projects/storefrontlib/src/lib/product/components/product-details/product-summary/product-summary.component.html

```diff
@@ -1,24 +1,42 @@
-<div class="cx-product-summary">
-  <div class="cx-product-summary__price cx-product-summary__price" aria-label="new item price">
-    {{product?.price?.formattedValue}}
-  </div>
-
-  <div class="cx-product-summary__description">
-    <p [innerHTML]="product?.summary"> </p>
-  </div>
+<div class="product-summary">
+  <ng-container *cxOutlet="outlets.TITLE">
+    <div class="item-title">
+      <div class="name">{{ product?.name }}</div>
+      <div class="code">ID {{ product?.code }}</div>
+    </div>
+  </ng-container>
 
-  <div class="cx-product-summary__variant">
-    <div class="cx-product-summary__variant-label">
-      Qty:
+  <ng-container *cxOutlet="outlets.RATING">
+    <div class="item-rating">
```

Comment:
np to have a wrapper div here. shall we simplify the name though? i.e. `rating`;
similarly we can make `review-count` just `count` and remove the `review-link` completely.

Reply:
Do you mean remove the class or the button?

Reply:
the class